qiskit_nature/algorithms/pes_samplers/bopes_sampler.py

```diff
@@ -152,9 +166,9 @@ def _run_points(self, points: List[float]) -> Dict[float, EigenstateResult]:
             The results for all points.
         """
         raw_results: Dict[float, EigenstateResult] = {}
-        if isinstance(self._gss.solver, VariationalAlgorithm):  # type: ignore
+        if isinstance(self._solver_wrapper.solver, VariationalAlgorithm):
```

Comment:
Given your comment on the main thread, I agree that storing `self._is_variational_solver` (or something like that) would make sense :+1: